features/seeds/seed_data.rb

```diff
@@ -23,10 +23,11 @@ def seed
 
   private
 
-  attr_reader :file_name
+  attr_reader :file_name, :options
 
-  def initialize(file_name)
+  def initialize(file_name, options)
```

Comment:
I'm struggling to understand how :options is called and used here. Is it possible to add more comments either here or in the README?

Reply:
This is not something that is `called`, is a parameter that is passed to the class for usage. This allow you to call the `seed` method passing a bunch of params after the file_name that will be stored in the object and used at need

Reply:
Thanks, also just spotted the following line:
`@reg_number = SeedData.seed("outstanding_balance_and_cards.json", copy_cards: copy_cards)`

What I'm getting at is: how do I know which parameters I can include when calling SeedData.seed, and what notation to use? To a developer this might be obvious, but to me or anyone else picking up the test suite, you'd need support to apply this. If it could be explained within the comments or README I'd appreciate it.

Reply:
Good point! Will add explanations in the readme :)